source/dub/compilers/compiler.d

```diff
@@ -8,7 +8,7 @@
 module dub.compilers.compiler;
 
 public import dub.compilers.buildsettings;
-public import dub.dependency : Dependency;
+deprecated(`Please "import dub.dependency : Dependency" instead.`) public import dub.dependency : Dependency;
```

Comment:
```suggestion
deprecated("Please `import dub.dependency : Dependency` instead") public import dub.dependency : Dependency;
```

This gives you syntax highlight, as DMD does for error messages.

Reply:
Interesting. Where is that documented?

Reply:
I don't think it's documented explicitly, but it follows from the fact that the error/diagnostic message printing logic in the frontend is shared. The same trick also applies to `static assert`.